addons/mail/static/src/core/web/discuss_sidebar_mailboxes.xml

```diff
@@ -16,12 +16,10 @@
     </t>
 
     <t t-name="mail.Mailbox">
-        <button
-            class="o-mail-DiscussSidebar-item btn d-flex align-items-center py-1 px-0 mx-2 border-0 rounded-0 fw-normal text-reset"
-            t-att-class="{
-                'bg-inherit': mailbox.notEq(store.discuss.thread),
-                'o-active': mailbox.eq(store.discuss.thread),
-            }"
+        <a
+            role="button"
+            class="o-mail-DiscussSidebar-item d-flex align-items-center py-1 px-0 mx-2 text-reset"
+            t-att-class="{ 'o-active': mailbox.eq(store.discuss.thread) }"
             t-on-click="(ev) => this.openThread(ev, mailbox)"
```

Comment:
valid `a` elements should always have `href`, and prevent propagation of click to avoid dummy `#` in url + scroll to top

Reply:
Or the best would be to actually have a properly working href based on router, so we don't need a click event at all? And allow to ctrl or middle click to open in new tab.

Reply:
You don't need a `href` for `a`, it's not mandatory. The role here is to specify that, in fact, the `a` is not a link to something, but a javascript action.  
https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#propri%C3%A9t%C3%A9s